File: Contracts/Model/Product/Response/ManagementProductsResponse.cs
namespace Contracts.Model.Product.Response;
public class ManagementProductsResponse : ByBaseResponse
{
	public Guid Id { get; set; }
	public string? Code { get; set; }
	public bool IsService { get; set; }
	public CategoriesListResponse Category { get; set; }
	public SubCategoriesListResponse? SubCategory { get; set; }
	public IEnumerable<OptionsListResponse> Options { get; set; }
}

File: ClientApp/src/app/core/interfaces/Product/Response/managementProductsResponse.ts
import {ByBaseResponse} from "../../byBaseResponse";
import {OptionsListResponse} from "./optionsListResponse";
import {CategoriesListResponse} from "@core/interfaces/Product/category";

export interface ManagementProductsResponse extends ByBaseResponse {
  id: string;
  code: string | null;
  isService: boolean;
  category: CategoriesListResponse;
  options: OptionsListResponse[];
}
